test/main.js

```diff
@@ -404,7 +404,7 @@ describe('node-fetch', () => {
 		const url = `${base}redirect/307`;
 		const options = {
 			method: 'PATCH',
-			body: resumer().queue('a=1').end()
+			body: stream.Readable.from('tada')
```

Comment:
Why did we go from resumer to a stream?

Reply:
One dependency less? `resumer` is unmaintained and never got out of version `0.0.0` while `Readable.from` is built-in Node.JS functions that doing the same.

Reply:
If the question is how the removal of `resumer` is relevant to this PR subject then the answer is easy - the "stream" that `resumer` is creating is not an instance of `Readable`.